app/src/db/models/tables/objectModel.js

```diff
@@ -62,6 +62,11 @@ class ObjectModel extends Timestamps(Model) {
       filterBucketIds(query, value) {
         filterOneOrMany(query, value, 'object.bucketId');
       },
+      filterName(query, value) {
+        if (value !== undefined) {
+          query.where('object.name', 'ilike', `%${value}%`);
+        }
```

Comment:
Shouldn't this follow the others and use the `filterILike` utils wrapper instead?